src/index.html

```diff
@@ -1,13 +1,55 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div
+    class="card"
+    data-qa="card"
+    >
+      <div class="card__image">
+        <img
+        class="iMac-image"
+        src="./images/imac.jpeg"
+        alt="iMac"
+      >
+      </div>
+      <div class="description card__description">
+        <div class="description__title">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </div>
+        <p class="description__code">Код товара: 195434</p>
+      </div>
+      <div class="rate card__rate">
+        <div class="rate__stars">
+          <span class="rate__stars-star"></span>
+          <span class="rate__stars-star"></span>
+          <span class="rate__stars-star"></span>
+          <span class="rate__stars-star"></span>
+          <span class="rate__stars-star"></span>
+        </div>
+        <p class="rate__feedback">Отзывов: 5</p>
+      </div>
+      <div class="price card__price">
```

Comment:
no need to make a separate BEM-block for price. It doesn't look like an independent logical block that can be reused